doc/api/async_hooks.md

```diff
@@ -460,6 +460,59 @@ init for PROMISE with id 6, trigger id: 5  # the Promise returned by then()
   after 6
 ```
 
+#### async_hooks.executionAsyncResource()
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} The resource that triggered the current
```

Comment:
I'm not sure about the word `triggered` here as it doesn't fit well to the doc of `triggerAsyncId` in [init hook](https://nodejs.org/dist/latest-v13.x/docs/api/async_hooks.html#async_hooks_init_asyncid_type_triggerasyncid_resource)
Maybe use `The resource representing the current execution`?

Reply:
Fixed.